inference-engine/tests/functional/plugin/cpu/single_layer_tests/group_convolution.cpp

```diff
@@ -347,7 +448,10 @@ const auto groupConvParams_ExplicitPadding_DW_3D = ::testing::Combine(
 const std::vector<CPUSpecificParams> CPUParams_DW_3D = {
         conv_sse42_dw_3D,
         conv_avx2_dw_3D,
-        conv_avx512_dw_3D
+        conv_avx512_dw_3D,
```

Comment:
Have you analyzed how functional tests time was affected by this changes? I just want to avoid huge impact on precommit performance.

Reply:
This PR adds a lot of convolution/group convolution test combinations. Not only for nspc layouts, but also for channel sizes that are not evenly divided by the block size, plain to blocked convolutions, etc. Of course, the number of test combinations has increased significantly. For example, before this PR the total number of convolution single layer test combinations was 2720, for this PR the number increased to 17120. The time for passing convolution tests is now 2 min 40 sec on CPX processor, before this PR it was 33 sec.

Reply:
Thats too much. Please consider decreasing number of test cases.

Reply:
The number of the test cases has been reduced by lowering the number of fused post ops. Now, on CPX machine we have 8732 test cases for convolutions, which take 81 sec, and 7252 test cases for group convolutions, which 72 sec.